<dec f='linux-4.18.y/include/linux/pci.h' l='715' type='4'/>
<doc f='linux-4.18.y/include/linux/pci.h' l='714'>/* Device has completely failed, is unrecoverable */</doc>
<use f='linux-4.18.y/drivers/dma/ioat/init.c' l='1262' u='r' c='ioat_pcie_error_slot_reset'/>
<use f='linux-4.18.y/drivers/net/ethernet/intel/e100.c' l='3087' u='r' c='e100_io_error_detected'/>
<use f='linux-4.18.y/drivers/net/ethernet/intel/e100.c' l='3110' u='r' c='e100_io_slot_reset'/>
<use f='linux-4.18.y/drivers/net/ethernet/intel/e1000/e1000_main.c' l='5259' u='r' c='e1000_io_error_detected'/>
<use f='linux-4.18.y/drivers/net/ethernet/intel/e1000/e1000_main.c' l='5291' u='r' c='e1000_io_slot_reset'/>
<use f='linux-4.18.y/drivers/net/ethernet/intel/e1000e/netdev.c' l='6806' u='r' c='e1000_io_error_detected'/>
<use f='linux-4.18.y/drivers/net/ethernet/intel/e1000e/netdev.c' l='6843' u='r' c='e1000_io_slot_reset'/>
<use f='linux-4.18.y/drivers/net/ethernet/intel/i40e/i40e_main.c' l='14203' u='r' c='i40e_pci_error_detected'/>
<use f='linux-4.18.y/drivers/net/ethernet/intel/i40e/i40e_main.c' l='14234' u='r' c='i40e_pci_error_slot_reset'/>
<use f='linux-4.18.y/drivers/net/ethernet/intel/i40e/i40e_main.c' l='14245' u='r' c='i40e_pci_error_slot_reset'/>
<use f='linux-4.18.y/drivers/net/ethernet/intel/igb/igb_main.c' l='8972' u='r' c='igb_io_error_detected'/>
<use f='linux-4.18.y/drivers/net/ethernet/intel/igb/igb_main.c' l='9000' u='r' c='igb_io_slot_reset'/>
<use f='linux-4.18.y/drivers/net/ethernet/intel/igbvf/netdev.c' l='2561' u='r' c='igbvf_io_error_detected'/>
<use f='linux-4.18.y/drivers/net/ethernet/intel/igbvf/netdev.c' l='2586' u='r' c='igbvf_io_slot_reset'/>
<use f='linux-4.18.y/drivers/net/ethernet/intel/ixgbe/ixgbe_main.c' l='10888' u='r' c='ixgbe_io_error_detected'/>
<use f='linux-4.18.y/drivers/net/ethernet/intel/ixgbe/ixgbe_main.c' l='10891' u='r' c='ixgbe_io_error_detected'/>
<use f='linux-4.18.y/drivers/net/ethernet/intel/ixgbe/ixgbe_main.c' l='10901' u='r' c='ixgbe_io_error_detected'/>
<use f='linux-4.18.y/drivers/net/ethernet/intel/ixgbe/ixgbe_main.c' l='10926' u='r' c='ixgbe_io_slot_reset'/>
<use f='linux-4.18.y/drivers/net/ethernet/intel/ixgbevf/ixgbevf_main.c' l='4770' u='r' c='ixgbevf_io_error_detected'/>
<use f='linux-4.18.y/drivers/net/ethernet/intel/ixgbevf/ixgbevf_main.c' l='4780' u='r' c='ixgbevf_io_error_detected'/>
<use f='linux-4.18.y/drivers/net/ethernet/intel/ixgbevf/ixgbevf_main.c' l='4806' u='r' c='ixgbevf_io_slot_reset'/>
<use f='linux-4.18.y/drivers/pci/pci-driver.c' l='1564' c='pci_uevent_ers'/>
<use f='linux-4.18.y/drivers/pci/pcie/err.c' l='41' c='merge_result'/>
<use f='linux-4.18.y/drivers/pci/pcie/err.c' l='207' u='r' c='reset_link'/>
<use f='linux-4.18.y/drivers/pci/pcie/err.c' l='213' u='r' c='reset_link'/>
<use f='linux-4.18.y/drivers/pci/pcie/err.c' l='328' u='r' c='pcie_do_fatal_recovery'/>
<use f='linux-4.18.y/drivers/pci/pcie/err.c' l='386' u='r' c='pcie_do_nonfatal_recovery'/>
<use f='linux-4.18.y/drivers/scsi/mpt3sas/mpt3sas_scsih.c' l='10911' u='r' c='scsih_pci_error_detected'/>
<use f='linux-4.18.y/drivers/scsi/mpt3sas/mpt3sas_scsih.c' l='10939' u='r' c='scsih_pci_slot_reset'/>
<use f='linux-4.18.y/drivers/scsi/mpt3sas/mpt3sas_scsih.c' l='10949' u='r' c='scsih_pci_slot_reset'/>
<use f='linux-4.18.y/drivers/tty/serial/8250/8250_pci.c' l='5176' u='r' c='serial8250_io_error_detected'/>
<use f='linux-4.18.y/drivers/tty/serial/8250/8250_pci.c' l='5193' u='r' c='serial8250_io_slot_reset'/>
<use f='linux-4.18.y/drivers/vfio/pci/vfio_pci.c' l='1275' u='r' c='vfio_pci_aer_err_detected'/>
<use f='linux-4.18.y/drivers/vfio/pci/vfio_pci.c' l='1280' u='r' c='vfio_pci_aer_err_detected'/>
